index.html

```diff
@@ -0,0 +1,21 @@
+<!doctype html>
```

Comment:
@vickywane why do we need to serve an html page?

Reply:
The page is what is built by Netlify on each deployment. I tried deploying a project without a build file but it breaks the deployment.